Build fails on Ubuntu 16.04

The default compiler on Xenial (gcc 5.4.0) does not enable c++11 features by default, therefore the build fails.

```
/home/martino/catkin_ws/src/kimera_rviz_markers/src/kimera_rviz_markers.cpp:13:7: error: expected nested-name-specifier before ‘IM’
 using IM = viz_msgs::InteractiveMarker;
       ^
/home/martino/catkin_ws/src/kimera_rviz_markers/src/kimera_rviz_markers.cpp:14:7: error: expected nested-name-specifier before ‘IMC’
 using IMC = viz_msgs::InteractiveMarkerControl;
       ^
make[2]: *** [CMakeFiles/kimera_rviz_markers.dir/src/kimera_rviz_markers.cpp.o] Error 1
make[1]: *** [CMakeFiles/kimera_rviz_markers.dir/all] Error 2
make: *** [all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(kimera_rviz_markers)
 
+set(CMAKE_CXX_STANDARD 11)
+
 find_package(catkin_simple REQUIRED)
 
 catkin_simple()

